src/kv/deserialise.h

```diff
@@ -623,23 +460,34 @@ namespace kv
         public_only_,
         v_,
         std::move(changes_),
-        std::move(new_maps_))
+        std::move(new_maps_)),
+      max_conflict_version(max_conflict_version_)
     {
       max_conflict_version = max_conflict_version_;
       tx = std::move(tx_);
     }
 
-    ApplyResult execute() override
+    ApplyResult apply() override
     {
-      return fn(tx, term, changes);
-    }
-
-    std::function<ApplyResult(
-      std::unique_ptr<Tx>& tx, Term term, OrderedChanges& changes)>
-      fn = [](std::unique_ptr<Tx>& tx, Term term, OrderedChanges& changes)
-      -> ApplyResult {
       tx->set_change_list(std::move(changes), term);
+
+      auto aft_requests = tx->rw<aft::RequestsMap>(ccf::Tables::AFT_REQUESTS);
+      auto req_v = aft_requests->get(0);
+      CCF_ASSERT(
+        req_v.has_value(),
+        "Deserialised append entry, but requests map is empty");
+      req = req_v.value();
+
       return ApplyResult::PASS;
-    };
+    }
+
+    kv::Version get_max_conflict_version() override
+    {
+      return max_conflict_version;
+    }
+
+  private:
+    uint64_t max_conflict_version;
+    std::unique_ptr<Tx> tx;
```

Comment:
Please move definition of these members to the top of the class rather than the bottom.